src/main/kotlin/com/openlattice/data/DataExpiration.kt

```diff
@@ -0,0 +1,22 @@
+package com.openlattice.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+import com.openlattice.edm.set.ExpirationType
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+/**
+ * Holds information about the expiration parameters of data within an entity set
+ */
+
+data class DataExpiration(
+        @JsonProperty(SerializationConstants.EXPIRATION) val timeToExpiration: Long,
```

Comment:
Please document the expected time unit here or make it a parameter.

Reply:
Also either here or in the ExpirationType class document the behavior of how expiration date is calculated based on the options. That will make it easier to suggests improvements as it is not clear exactly how timeToExpiration interacts (my guess based on the naming is baseTime + timeToExpiration). If you are using it as the base for an offset considering renaming it ExpirationBase instead of ExpirationType, which may conflict with or scheme with names and types.

Reply:
I changed the name to ExpirationBase, you're right that's much clearer! And I fleshed out the documentation, hopefully it is clear now.

Reply:
This is great. One last minor change-- use the parameter synax. i.e @param before variable names in documentation

Reply:
added!